src/js/components/ClusterDropdown.js

```diff
@@ -0,0 +1,181 @@
+import * as React from "react";
+import mixin from "reactjs-mixin";
+import { Dropdown } from "reactjs-components";
+import { StoreMixin } from "mesosphere-shared-reactjs";
```

Comment:
I thought we were not supposed to be using mixins anymore. Can you check please @DanielMSchmidt

Reply:
That is one of the things I meant in the sync, it is just copied over logic and I don't really think removing the store mixin here and moving everything away from the stores, in general, is part of this ticket 🙈

Reply:
As discussed offline. Let's try just replacing the mixing with empty subscriptions. The component doesn't have any loading state etc. it just uses the data without prior checking whether it has been loaded and unsubscribes after the first response.